src/components/POLine/Cost/CostForm.js

```diff
@@ -101,6 +104,10 @@ class CostForm extends Component {
     const discountType = get(formValues, 'cost.discountType', DISCOUNT_TYPE.amount) || DISCOUNT_TYPE.amount;
     const isAmountDiscountType = discountType === DISCOUNT_TYPE.amount;
     const poLineEstimatedPrice = calculateEstimatedPrice(formValues);
+    const DEFAULT_CURRENCY_OPTIONS = [{ label: 'USD', value: 'USD' }];
+    const currenciesOptions = currencies.length
```

Comment:
`currencies` might be undefined, as I could see.

Reply:
currencies is field of vendor object and it's value at least empty array